### PR TITLE
[FLINK-25789][docs-zh] Translate the formats/hadoop page into Chinese.

### PR DESCRIPTION
**Pull request by RocMarshal**
## What is the purpose of the change

*Translate the formats/hadoop page into Chinese.*

## Brief change log

Translate the formats/hadoop page into Chinese.
file: location: docs/content.zh/docs/connectors/datastream/formats/hadoop.md

## Verifying this change

a pure documentation translation task.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by RocMarshal** [comment]
Hi, @Thesharing , May I get your help to review it?  Thank you.

File: docs/content.zh/docs/connectors/datastream/formats/hadoop.md [hunk 1]
```diff
@@ -57,20 +55,13 @@ a `hadoop-client` dependency such as:
...
 
-To use Hadoop `InputFormats` with Flink the format must first be wrapped
-using either `readHadoopFile` or `createHadoopInput` of the
-`HadoopInputs` utility class.
-The former is used for input formats derived
-from `FileInputFormat` while the latter has to be used for general purpose
-input formats.
-The resulting `InputFormat` can be used to create a data source by using
-`ExecutionEnvironmen#createInput`.
+要将 Hadoop `InputFormats` 与 Flink 一起使用，必须首先使用 `HadoopInputs` 工具类的 `readHadoopFile` 或 `createHadoopInput` 包装格式。
```
Comment (Thesharing) [inline docs/content.zh/docs/connectors/datastream/formats/hadoop.md:58]:
```suggestion
要在 Flink 中使用 Hadoop `InputFormats`，必须首先使用 `HadoopInputs` 工具类的 `readHadoopFile` 或 `createHadoopInput` 包装 Input Format。
```

File: docs/content.zh/docs/connectors/datastream/formats/hadoop.md [hunk 2]
```diff
@@ -57,20 +55,13 @@ a `hadoop-client` dependency such as:
...
+前者用于从 `FileInputFormat` 派生的输入格式，而后者必须用于通用输入格式。
```
Comment (Thesharing) [inline docs/content.zh/docs/connectors/datastream/formats/hadoop.md:59]:
```suggestion
前者用于从 `FileInputFormat` 派生的 Input Format，而后者必须用于通用的 Input Format。
```

**Review by Thesharing — COMMENTED** [review]
Thank you for contributing this translation to the Flink community. Your translation is great. I believe your translation will help Chinese Flink users understand the docs better. I have two minor suggestions:

1. "输入格式" is not widely used to translate `Input Format`. It may confuse readers because they don't know what "输入格式" is. Maybe leaving it untranslated is better.
2. In the original text, sometimes "InputFormat" is wrapped with \`\`, sometimes it is not. Maybe we should keep it consistent. All `InputFormat` and `OutputFormat` should be wrapped with \`\`.

These suggestions may not be applicable. Feel free to have a discussion with me. By the way, since I'm not a Flink committer, I don't have the privilege to merge this pull request.

**Comment by RocMarshal** [comment]
@Thesharing Thank you very much for the comments.  Updated it.

File: docs/content.zh/docs/connectors/datastream/formats/hadoop.md [hunk 3]
```diff
@@ -57,20 +55,13 @@ a `hadoop-client` dependency such as:
...
 
-To use Hadoop `InputFormats` with Flink the format must first be wrapped
-using either `readHadoopFile` or `createHadoopInput` of the
-`HadoopInputs` utility class.
-The former is used for input formats derived
-from `FileInputFormat` while the latter has to be used for general purpose
-input formats.
-The resulting `InputFormat` can be used to create a data source by using
-`ExecutionEnvironmen#createInput`.
+要在 Flink 中使用 Hadoop `InputFormats`，必须首先使用 `HadoopInputs` 工具类的 `readHadoopFile` 或 `createHadoopInput` 包装 Input Format。
```
Comment (RocMarshal) [inline docs/content.zh/docs/connectors/datastream/formats/hadoop.md:58]:
```suggestion
在 Flink 中使用 Hadoop `InputFormats`，必须首先使用 `HadoopInputs` 工具类的 `readHadoopFile` 或 `createHadoopInput` 包装 Input Format。
```
```suggestion
要在 Flink 中使用 Hadoop `InputFormats`，必须首先使用 `HadoopInputs` 工具类的 `readHadoopFile` 或 `createHadoopInput` 包装 Input Format。
```

**Comment by RocMarshal** [comment]
@leonardBang Could you help me to merge it if there's nothing Inappropriate？  Thank you very much.

**Comment by RocMarshal** [comment]
@flinkbot run azure

**Review by gaoyunhaii — APPROVED** [review]
LGTM, thanks @RocMarshal for the translating and @Thesharing for the review!

**Pull request closed without merging**